go.mod

```diff
@@ -174,7 +174,7 @@ require (
 	github.com/elastic/bayeux v1.0.5
 	github.com/elastic/ebpfevents v0.6.0
 	github.com/elastic/elastic-agent-autodiscover v0.9.0
-	github.com/elastic/elastic-agent-libs v0.19.0
+	github.com/elastic/elastic-agent-libs v0.19.1-0.20250410071650-d752da92e87e
```

Comment:
[Blocker]
Shouldn't you properly release `github.com/elastic/elastic-agent-libs` first?